labs/smartbft.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Hyperledger Fabric SmartBFT Library
+parent: Labs
+
+---
+# Lab Name
+
+[SmartBFT] - https://github.com/SmartBFT-Go/consensus
+
+
+# Short Description
+
+Hyperledger Fabric is an enterprise-grade, permissioned distributed ledger platform that offers modularity for a broad set of industry use cases. One modular component is a pluggable ordering service that establishes consensus on the order of transactions and batches them into blocks. This proposal suggests integrating the SmartBFT consensus library into Hyperledger Labs to enhance its blockchain frameworks with a robust, scalable, and resilient consensus mechanism designed for practical BFT deployment.
+
+This lab provides a BFT consensus library based on the [BFTSmart](https://github.com/bft-smart/library) protocol and written in Go. The library is tailored to the blockchain use case yet general enough to cater to a wide variety of other uses. The BFT library's design and integration into Fabric address crucial aspects that were left unsolved in all prior work, making it unfit for production use. 
```

Comment:
"making it unfit for production use" - Not sure if this implies to the code as it exists now or the code that it was based on.

Reply:
I think it refers to previous works :-)

Reply:
It actually refers to this work: https://github.com/bft-smart/library, which was initially done. However, it wasn't adopted because it doesn't really fit production real-world use cases.

Reply:
fix